app/assets/javascripts/Resources/Address.coffee

```diff
@@ -0,0 +1,8 @@
+app = angular.module('customerApp')
```

Comment:
omg. don't put capital letters in your file names! WHAT IS THIS ANARCHY!? underscore_case_always_unless_its_a_javascriptVariableOrRUBY_CONSTANT